src/PKSim.Core/Model/ContainerParameterBySpecies.cs

```diff
@@ -0,0 +1,9 @@
+﻿namespace PKSim.Core.Model
+{
+   public class ContainerParameterBySpecies
+   {
+      public string ContainerPath { get; set; }
+      public string ParameterName { get; set; }
+      public int SpeciesCount { get; set; }
+   }
+}
```

Comment:
Added the new type (did not find a good name - if you can think of a better name - let me know :))